Code/CSharpOOP3/Helper.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace CSharpOOP3
+{
+    public class Helper
```

Comment:
this name is not really meaningful :(
there could be a lot of helpers in your project. Please rename to smth like FlowerHelper

Reply:
done